eng/pipelines/templates/jobs/ci.yml

```diff
@@ -191,6 +190,55 @@ jobs:
           failOnStderr: false
       - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
 
+  - job: Compliance
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+    steps:      
+      - pwsh: |
+          New-Item -Path "$(Build.sourcesdirectory)" -Name "credscan.tsv"
+          if ("$(Build.Reason)" -eq 'PullRequest') {
+            New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+            (git diff "origin/$(System.PullRequest.TargetBranch)" HEAD --name-only | Split-Path)
+            | ForEach-Object { Set-Content -Path "$(Build.SourcesDirectory)\credscan.tsv" -Value $_}
```

Comment:
This looks like it will only write the last directory to the file because it keeps overwritting the file.